prisma/schema.prisma

```diff
@@ -165,6 +176,27 @@ model TrackCollaborator {
     role            TrackCollaboratorRole
 }
 
+model TrackAuditLog {
+    id                    String              @id @default(cuid())
+    track                 Track               @relation(fields: [trackId], references: [id], onDelete: Cascade)
+    trackId               String
+    user                  User?               @relation("User", fields: [userId], references: [id])
+    userId                String?
+    discordUserId         String?
+    discordUsername       String?
+    discordAvatar         String?
+    targetUser            User?               @relation("TargetUser", fields: [targetUserId], references: [id])
+    targetUserId          String?
+    targetDiscordUserId   String?
+    targetDiscordUsername String?
+    targetDiscordAvatar   String?
+    action                TrackAuditLogAction
+    value                 String?
+    oldValue              String?
```

Comment:
Can value be expanded to a polymorphic relationship?

Reply:
or a JSON column since we're in postgres

Reply:
Made it Json